rust_src/src/frames.rs

```diff
@@ -593,4 +594,22 @@ pub fn frame_focus(frame: LispFrameLiveOrSelected) -> LispObject {
     frame_ref.focus_frame
 }
 
+/// Set the input focus to FRAME.
+/// FRAME nil means use the selected frame.  Optional argument NOACTIVATE
+/// means do not activate FRAME.
+/// If there is no window system support, this function does nothing.
```

Comment:
There should be an empty line with just `///` above the line 'If there is no....'.